src/app/footer/footer.component.html

```diff
@@ -0,0 +1,89 @@
+<div class="fixed-bottom mt-2">
+
+  <div *ngIf="game && (activeTurn === 1 || activeTurn === 2 || activeTurn === 3 || activeTurn === 4 ) "
```

Comment:
```suggestion
  <div *ngIf="game && [1,2,3,4].includes(activeTurn)"
```
or consider making a getter